src/integrationTest/java/uk/gov/hmcts/reform/professionalapi/util/ProfessionalReferenceDataClient.java

```diff
@@ -20,9 +20,18 @@
 
 public class ProfessionalReferenceDataClient {
 
-    private static final String APP_BASE_PATH = "v1/organisations";
+    private static final String APP_BASE_PATH = "refdata/external/v1/organisations";
```

Comment:
Change the name to be more descriptive e.g. APP_BASE_PATH_EXT and APP_BASE_PATH_INT.

Reply:
done